src/index.test.ts

```diff
@@ -54,26 +53,19 @@ const address = '0x0c03fba782b07bcf810deb3b7f0595024a444f4e';
 
 describe('Running @erc725/erc725.js tests...', () => {
   it('should throw when no arguments are supplied', () => {
-    assert.throws(
-      () => {
-        // @ts-ignore
-        // eslint-disable-next-line no-new
-        new ERC725();
-      },
-      (error: any) => error.message === 'Missing schema.',
-    );
+    assert.throws(() => {
+      // @ts-ignore
+      // eslint-disable-next-line no-new
+      new ERC725();
+    }, 'Missing schema.');
   });
 
   it('should throw when incorrect or unsupported provider is provided', () => {
-    assert.throws(
-      () => {
-        // @ts-ignore
-        // eslint-disable-next-line no-new
-        new ERC725(mockSchema, address, { test: false });
-      },
-      (error: any) =>
-        error.message.indexOf('Incorrect or unsupported provider') >= -1,
-    );
+    assert.throws(() => {
+      // @ts-ignore
+      // eslint-disable-next-line no-new
+      new ERC725(mockSchema, address, { test: false });
+    }, /Incorrect or unsupported provider/);
```

Comment:
NIce it's more readable.
Can't we put the error as a string tho?

```suggestion
    }, "Incorrect or unsupported provider");
```

Reply:
I suppose we could. I just converted the original code to the new one leaving the same expression.